src/examples/button/SplitButtonSizes.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { SplitButton, Button, ChevronButton } from '@zendeskgarden/react-buttons';
+import { Dropdown, Trigger, Menu, Item } from '@zendeskgarden/react-dropdowns';
+
+const Example = () => (
+  <Row>
+    <Col>
```

Comment:
Add `textAlign="center"` here.

Reply:
All `Col` elements should have a `sm` boolean prop in this example to support an improved layout for mobile. Also, 2nd and 3rd columns should be styled with a responsive margin. See the [Inputs size](https://5ef13c3303616c01cc6490d3--zendeskgarden.netlify.app/components/inputs#size) example for details.